packages/netlify-plugin-cache-docs/index.js

```diff
@@ -59,8 +59,8 @@ module.exports = {
     const cacheManifestPath = path.join(PUBLISH_DIR, cacheManifestFileName);
     console.log('Saving cache file manifest for debugging...');
     const files = await utils.cache.list();
-    await fse.mkdirp(PUBLISH_DIR);
-    await fse.writeJSON(cacheManifestPath, files, { spaces: 2 });
+    await fs.promises.mkdir(PUBLISH_DIR, { recursive: true });
```

Comment:
Only using `fs.promises` while we depend on sync methods. Once we get rid of `existsSync`, I'll update the import to be `fs/promises` and we can refactor this.